src/assets/images/list.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No need to say (not great) in the commit message, perhaps something like:
`Added intial draft of icon for lists`